scripts/test-cli-integration

```diff
@@ -37,7 +38,7 @@ EOF
 
 jq -c '.' ${CLIENT_BUCKET}/data.ndjson
 
-python -m prio encode-shares \
+python3 -m prio encode-shares \
```

Comment:
Is the python3 command necessary?

Reply:
Yes, unfortunately python3 is not the default python in the Fedora or Ubuntu images.